srv/methods.go

```diff
@@ -148,7 +171,10 @@ type ResultGetTransaction struct {
 	Tx        interface{}     `json:"actions"`
 }
 
+// getTransaction is deprecated
 func (s *APIServer) getTransaction(getEntry bool) jrpc.MethodFunc {
+	// TODO: Should we just remove this?
```

Comment:
rename it to "decode-tx-entry"?

Reply:
I'm more in favor of just dropping it. I don't even know if the functionality matches the get-transaction

Reply:
i'm okay with dropping it too